lib/blobs.js

```
@@ -0,0 +1,30 @@
+/**
+ * @typedef {{
+ * blobId: string,
+ * blobIdBase64: string,
+ * objectId: string,
+ * status: string,
+ * startEpoch: number,
+ * endEpoch: number,
+ * size: number,
+ * timestamp: number
+ * }} Blob
+ */
+
+/**
+ * Returns a list of recent confirmed blobs.
+ *
+ * @param {typeof globalThis.fetch} fetch The fetch function to use.
+ * @returns {Promise<string[]>} A list of recent blob Base64 IDs.
+ */
+export const getRecentBlobs = async (fetch = globalThis.fetch) => {
+  const res = await fetch('https://walruscan.com/api/walscan-backend/testnet/api/blobs?page=0&sortBy=TIMESTAMP&orderBy=DESC&searchStr=&size=20')
```

Are we sure we do not want to wrap this in a try catch block?

---

Since we're calling `getRecentBlobs` from try/catch block in the main loop maybe we should let it fail and let error be logged. WDYT?